px-app-nav.es6.js

```diff
@@ -480,17 +505,17 @@
      * will not be triggered.
      */
     _handleResize(evt) {
-      if (this.collapseAll || this.vertical) return;
+      if (this.collapseAll || (this.vertical && typeof this.verticalExpandedAt !== 'number')) return;
```

Comment:
All the type checks everywhere feel weird. What's their purpose (vs checking truthiness)?

Reply:
I'm just jumping on the bandwagon, but I think it was done that way in order to cover the most ground: not null **and** a number.

Reply:
Yeah you're right, it seems a little over-baked to me as well but I remember I did run into a scenario during debugging where `_availableWidth` for instance was typed as `object` for some reason, so I decided to stick with the convention.